dashboard/v1.1/src/services/input.ts

```diff
@@ -0,0 +1,42 @@
+export const defaultHTTPMethodsList = [
+  {
+    name: 'HTTP',
+    value: 'HTTP'
+  },
+  {
+    name: 'HTTPS',
+    value: 'HTTPS'
+  },
+  {
+    name: 'Manual',
+    value: 'Manual'
+  },
+];
+
+export const defaultRequestMethodsList = [
+  {
+    name: 'GET',
+    value: 'GET'
+  },
+  {
+    name: 'POST',
+    value: 'POST'
+  },
+  {
+    name: 'PUT',
+    value: 'PUT'
+  },
+  {
+    name: 'DELETE',
+    value: 'DELETE'
```

Comment:
These should be `enum`.

Reply:
we extend to a common type for all the dropdowns which expects it to be a `name, value` format. Hence doing it as enum will increase the type definitions and it is hard to edit if we shifted later to more url request types which aren't as simple as `HTTP: HTTP`